src/integration/transaction/api/controllers/transaction.controller.ts

```diff
@@ -40,7 +40,7 @@ export class TransactionController {
   @Put(':id/signed')
   @ApiBearerAuth()
   @ApiExcludeEndpoint()
-  @UseGuards(AuthGuard(), new RoleGuard(WalletRole.TRANSACTION_SIGNER))
+  @UseGuards(AuthGuard(), new RoleGuard(WalletRole.MASTERNODE_MANAGER))
```

Comment:
Why revert?

Reply:
Because the signing is done by the cold-wallet and therefore masternode manager need to be allowed to updated a signed transaction.

Reply:
IMO we should only have two staking roles TRANSACTION_CHECKER and TRANSACTION_SIGNER. Am I missing something?

Reply:
ah we are talking about two different things 😂 We currently have TRANSACTION_SIGNER and MASTERNODE_MANAGER. Then I f'd up another change. As I have thought that TRANSACTION_CHECKER should be renamed to TRANSACTION_SIGNER

Reply:
Ahhh ok 🤣